executor/analyze_test.go

```diff
@@ -1233,3 +1233,256 @@ func TestAnalyzeSamplingWorkPanic(t *testing.T) {
 	require.NotNil(t, err)
 	require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/executor/mockAnalyzeSamplingMergeWorkerPanic"))
 }
+
+func TestSavedAnalyzeOptions(t *testing.T) {
+	store, clean := testkit.CreateMockStore(t)
```

Comment:
We can use `CreateMockStoreAndDomain` to get `dom`. Then we can get `h` and `is` by `dom.StatsHandle()` and `dom.InfoSchema()` respectively.

Reply:
Done